src/modules/Laboratorio/Listar/actions/selectionStatus.js

```diff
@@ -1,12 +1,12 @@
 module.exports = function selectionStatus(status) {
-  if (status === 'Livre') {
+  if (status === false) {
     return 'laboratorioStatusVerde';
   }
-  if (status === 'Ocupado') {
+  if (status === true) {
     return 'laboratorioStatusVermelho';
   }
-  if (status === 'Reservado') {
-    return 'laboratorioStatusAmarelo';
-  }
+  //if (status === 'Reservado') {
+  //  return 'laboratorioStatusAmarelo';
+  //}
   return 'laboratorioStatusError';
```

Comment:
Esse trecho de código é inalcançável se for realmente utilizado uma variável `boolean`. Então pode ser removido.

```suggestion
```